deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -382,14 +380,10 @@ data:
       log_level info
     </system>
   
-  buffer.output.conf: |-
-    compress gzip
-    flush_interval "#{ENV['FLUSH_INTERVAL']}"
-    flush_thread_count "#{ENV['NUM_THREADS']}"
-    chunk_limit_size "#{ENV['CHUNK_LIMIT_SIZE']}"
-    total_limit_size "#{ENV['TOTAL_LIMIT_SIZE']}"
-    queued_chunks_limit_size "#{ENV['QUEUE_CHUNK_LIMIT_SIZE']}"
-    overflow_action drop_oldest_chunk
+  buffer.output.conf: "compress gzip\nflush_interval \"#{ENV['FLUSH_INTERVAL']}\"\nflush_thread_count
+    \"#{ENV['NUM_THREADS']}\"\nchunk_limit_size \"#{ENV['CHUNK_LIMIT_SIZE']}\"\ntotal_limit_size
+    \"#{ENV['TOTAL_LIMIT_SIZE']}\"\nqueued_chunks_limit_size \"#{ENV['QUEUE_CHUNK_LIMIT_SIZE']}\"\noverflow_action
+    drop_oldest_chunk "
```

Comment:
I think previous file was generated more readable

Reply:
Yeah, I wonder what happened 🤔

Reply:
looks line it was the new line

Reply:
Yes for future reference any trailing whitespace at the end of a line can cause helm templating to go crazy

Reply:
👍 Fixed :)